client/src/modules/depots/modals/depot.modal.js

```diff
@@ -19,6 +19,13 @@ function DepotModalController($state, Depots, ModalService, Notify) {
   function submit(depotForm) {
     var promise;
 
+    var stockEntryCache = new AppCache('StockEntry');
+    var stockExitCache = new AppCache('StockExit');
+
+    // remove depot from caches
+    delete stockEntryCache.depot;
```

Comment:
It looks like this is a hack to make sure old cache values are not preserved.  The best way to do this is always cache only the id of the depot ... that way, the depot will be redownloaded each time and the new text will be presented.